.github/workflows/snapcraft-publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Snapcraft Publish Action
```

Comment:
Rename this file to `deploy-linux` and create another action which only builds the linux package but does not upload it. This should run on pull request.